Add deletion protection docs to VSHN postgres (#56)
vshn/appcat-user-docs: docs/modules/ROOT/pages/vshn-managed/postgresql/deletion-protection.adoc

= Database Deletion Protection

[IMPORTANT]
====

> Deletion Protection feature is enabled by default and insures backups are available 7 days
> after a database instance is deleted. Any backup available at the time of an instance deletion can be used
> to restore data into a new instance. The Deletion Protection feature does not restore the deleted
> database instance.

Reviewer: ```suggestion
The Deletion Protection feature is enabled by default and ensures backups are available for 7 days
after a database instance is deleted. Any backup available during an instance deletion can be used
to restore data into a new instance. The Deletion Protection feature does not restore the deleted
database instance.
```

Reviewer: Sure, as you wish
